### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by bohdan-mykhailenko**
[DEMO LINK](https://bohdan-mykhailenko.github.io/react_goods-selector/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +16,82 @@ export const goods = [
...
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+              />
+            </h1>
+          ) : (
+            <h1 className="title">
+              No goods selected
+            </h1>
+          )}
```
Comment (naziamoff) [inline src/App.tsx:50]: I would say it fits into a single line perfectly
```suggestion
          ) 
        : <h1 className="title">No goods selected</h1>}
```

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +16,82 @@ export const goods = [
...
+                  {
+                    'has-background-success-light': good === selectedGood,
+                  },
+                )}
+              >
+                <td>
+                  {good === selectedGood
+                    ? (
+                      <button
+                        onClick={() => this.setState({ selectedGood: '' })}
```
Comment (naziamoff) [inline src/App.tsx:68]: create a method for that

File: src/App.tsx [hunk 3]
```diff
@@ -15,71 +16,82 @@ export const goods = [
...
+                        data-cy="RemoveButton"
+                        type="button"
+                        className="button"
+                      >
+                        -
+                      </button>
+                    ) : (
+                      <button
+                        onClick={() => this.setState({ selectedGood: good })}
```
Comment (naziamoff) [inline src/App.tsx:77]: create a method for that

**Review by naziamoff — CHANGES_REQUESTED** [review]
not bad, but some changes requested

File: src/App.tsx [hunk 4]
```diff
@@ -15,71 +16,86 @@ export const goods = [
...
+              <button
+                onClick={this.removeSelectedGood}
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+              />
+            </h1>
+          ) : (
+            <h1 className="title">No goods selected</h1>
+          )}
```
Comment (33Orange) [inline src/App.tsx:56]: Check @naziamoff comment please
```suggestion
          ) : <h1 className="title">No goods selected</h1>}
```

File: src/App.tsx [hunk 5]
```diff
@@ -15,71 +16,86 @@ export const goods = [
...
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+        <table className="table">
+          <tbody>
+            {goods.map((good) => (
+              <tr
```
Comment (33Orange) [inline src/App.tsx:61]: Let's create a variable for this condition

```suggestion
            {goods.map((good) => (
              const isSelected = good === selectedGood;
          
              return (<tr
```

**Review by 33Orange — CHANGES_REQUESTED** [review]
Almost done

**Review by 33Orange — APPROVED** [review]
Well done

**Pull request closed without merging**